self-hosted-login/okta-aspnetcore-mvc-example/Startup.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Okta.AspNetCore;
+
+namespace okta_aspnetcore_mvc_example
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var oktaMvcOptions = new OktaMvcOptions();
+            Configuration.GetSection("Okta").Bind(oktaMvcOptions);
```

Comment:
Bind is cool, but I would prefer to be more explicit, like:

```
new OktaMvcOptions
{
  ClientId = Configuration["Okta:ClientId"],
  ...
}
```

That way it is really clear where the value is coming from.